src/main/java/com/leetcodeprep/linkedList/LinkedListCustom.java

```diff
@@ -0,0 +1,77 @@
+package com.leetcodeprep.linkedList;
```

Comment:
It is common practice to use Google Java code convention. It gives possibility to have the same code and the same rules for everyone.
Also there is excellent tool Spotless, which can make formatting automatically and also according to Google convention.

For example, according to one, package names should be lower-case. https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

And also whole code should be formatted at least as Idea formatter can do it. 
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html